src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple iMac Computer"
+        class="card__image"
+      >
+      <div class="card__description">
+        <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__product-code">Product code: 195434</p>
+      </div>
+      <div class="card__review">
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="Apple iMac Computer"
        class="card__image"
      >
      
      <div class="card__description">
        <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
        <p class="card__product-code">Product code: 195434</p>
      </div>
      
      <div class="card__review">
```
checklist 
<img width="920" alt="image" src="https://user-images.githubusercontent.com/102411130/226280694-cc4ce6ef-3af5-4970-9028-75e58e8db681.png">